ecs/ecs.go

```diff
@@ -122,17 +122,31 @@ func (c *Client) GetResourceGraph() (ResourceNode, error) {
 			return ResourceNode{}, err
 		}
 
-		describeTasksInput := &ecs.DescribeTasksInput{
-			Cluster: aws.String(clusterName),
-			Tasks:   taskARNs,
+		idChunks := [][]*string{}
```

Comment:
add comment on this block, or could even be a function since this is an isolated operation not really interacting with any other things in this flow?